src/axom/slic/docs/sphinx/sections/appendix.rst

```diff
@@ -52,6 +52,66 @@ logging within an application.
   functionality in application specific macros to better suit the requirements
   of the application.
 
+.. _CollectiveSlicMacros:
+
+Collective Slic Macros
+^^^^^^^^^^^^^^^^^^^^^^^^^
+A subset of SLIC macros are collective operations when used with
+MPI-aware :ref:`LogStream` instances such as :ref:`SynchronizedStream`
+or :ref:`LumberjackStream`.
+
+Additionally, macros such as ``SLIC_WARNING`` and ``SLIC_CHECK`` become collective
+operations when certain flags are toggled on or functions are called. Other macros
+such as ``SLIC_ERROR`` and ``SLIC_ASSERT`` can be made not collective when certain
+functions are called.
+
+The table below details which SLIC macros are collective:
+
+.. list-table:: Macros, Collective and Non-Collective
+   :widths: 20 10 70
+   :header-rows: 1
+
+   * - Macro
```

Comment:
Nice table!

Since consecutive entries, e.g.  `SLIC_INFO` and `SLIC_INFO_IF` have the same values, would it make sense to combine these into a single row?

E.g. like this: https://stackoverflow.com/a/34222391